crit/Makefile

```diff
@@ -0,0 +1,27 @@
+GO ?= go
+
+# The import path that protoc will use if a proto file imports another one
+import_path := github.com/checkpoint-restore/go-criu/crit/images
+# Path to .proto source files
+proto_path := ./images
+# Generate string of all .proto filenames
+proto_files := $(sort $(subst $(proto_path)/,,$(wildcard $(proto_path)/*.proto)))
+# Generate M flag to specify import path for all .proto files
+# and replace all spaces with commas to use with go_opt flag
+comma := ,
+proto_opts := $(subst $() $(),$(comma),$(patsubst %,M%=$(import_path),$(proto_files)))
+
+get-proto:
```

Comment:
> It would be good to commit crit/images/*.pb.go files.

The original `criu` repo maintains only the `.proto` files, and uses the Makefile to generate the bindings for both Python and C. I thought it would be better to stick to the same system here. I have added the target to the parent Makefile [here](https://github.com/checkpoint-restore/go-criu/pull/66/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R17), which also ensures that the tests pass. I'm unsure if there are any other benefits to committing the `.pb.go` files.

Reply:
I've added `crit/images` to `skip-dirs` in `.golangci.yml`. I just noticed that there was a typo that stopped it from working. The YAML section should be `skip-dirs` with a hyphen, not underscore. This should fix the lint errors.